modules/DefaultRenderer.js

```diff
@@ -136,13 +136,14 @@ class Navigation extends Component<void, Props, void> {
 
   render(): React$Element<any> {
     const { navigationState } = this.props
+    const defualtRender = (ownProps) => this.renderView({ ...this.props, ...ownProps })
     return (
       <Transitioner
-        configureTransition={this.configureTransition}
+        configureTransition={this.props.configureTransition || this.configureTransition}
```

Comment:
It would be so great to get this option merged into the next RC. I have a hack in place to inject this option for my app. Thanks!